geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartGatewaySenderCommand.java

```diff
@@ -77,6 +77,49 @@ public ResultModel startGatewaySender(@CliOption(key = CliStrings.START_GATEWAYS
       return ResultModel.createError(CliStrings.NO_MEMBERS_FOUND_MESSAGE);
     }
 
+    if (cleanQueues) {
+
+      Set<DistributedMember> allServers = findMembers(null, null);
+
+      if (dsMembers.size() != allServers.size()) {
+        return ResultModel.createError(CliStrings.EXECUTE_ON_ALL_MEMBERS);
+      }
```

Comment:
This code will reject a start clean queue command when you execute it for a particular group of servers when there are multiple groups of servers. Could it be possible to check that only servers in the provided group have configured senders and continue?

Reply:
Thanks for comment. It seems if partitioned region is created on just a group of servers, this check is not valid.